acmw-db-app/pages/api/account/[pid].js

```diff
@@ -32,9 +32,12 @@ async function generateToken (email) {
 async function checkReset (token) {
     // Get the email address associated with a token if token is not expired
     const validResetWindow = await pgQuery(`
-        SELECT email FROM account 
+        SELECT email FROM account
         WHERE current_timestamp <= token_expire_time AND reset_token = '${token}';
     `);
+  
+    if(validResetWindow.rowCount === 0) throw ('Reset password link either expired or invalid');
```

Comment:
delete this please, ik you didn't put it there originally but the SHA number diffs make me worry